### DISCUSSION TITLE
$error: What is the same structure in Python?

### DISCUSSION THREAD

**Discussion by whyImeow** [cocotb/cocotb#3004] [Q&A]
I always use a wonderful function of $error in SystemVerilog which doesn't only emit an error severity message, but also shows a place with an error in waveform by red triangles (in QuestaSim simulator). BUT! I didn't find something similar in cocotb. 
To attain this in SystemVerilog tests, I had to write only this:
`$info("passed"); else $error("failed");`
But what should I do if I want to attain such result in cocotb?
![error](https://user-images.githubusercontent.com/106060158/169804393-b48642c5-fcfb-4db4-9a20-48aebd1d26d1.PNG)

**Comment by ktbarrett** [comment]
There is no exactly equivalent in cocotb. cocotb is not tightly integrated into the simulator like the SystemVerilog functionality is. However, if you are in the GUI, the timestamps that cocotb log messages print can be clicked on, and it will move the waveform cursor to that timestamp.
